.dagger/go.mod

```diff
@@ -7,6 +7,7 @@ require github.com/dagger/dagger/engine/distconsts v0.14.0
 replace github.com/dagger/dagger/engine/distconsts => ../engine/distconsts
 
 require (
+	dagger.io/dagger v0.14.0
```

Comment:
:eyes: this shouldn't be here - the `dagger` module we want should be created by `dagger develop`.

Reply:
Oh, I don't know why it added. 

Reverted.

Reply:
Ah woops, this was caused by https://github.com/dagger/dagger/commit/6fdf164c7f62a8e57d6306ba07692feef1c04501 - the import added to `sdk_elixir.go` was wrong, you removed the telemetry dependency so it's resolved now.